agentstack/cli/cli.py

```diff
@@ -38,21 +90,16 @@ def init_project_builder(slug_name: Optional[str] = None, skip_wizard: bool = Fa
         }
 
         tools = []
-    else:
-        welcome_message()
-        project_details = ask_project_details(slug_name)
-        welcome_message()
-        framework = ask_framework()
-        design = ask_design()
-        tools = ask_tools()
 
     log.debug(
```

Comment:
I ❤️ logging for the realtime visibility into stuff happening in the background. We should think of it like a tool for the users, too.

Reply:
definitely agree. maybe we can make it more accessible too by changing the logging level if they use a `--verbose` flag on any command